shell/src/main/java/alluxio/cli/fsadmin/report/CapacityCommand.java

```diff
@@ -222,6 +278,38 @@ private GetWorkerReportOptions getOptions(CommandLine cl) throws IOException {
     return workerOptions;
   }
 
+  /**
+   * Gets the formatted tier values of a worker.
+   *
+   * @param map the map to get worker tier values from
+   * @param workerName name of the worker
+   * @return the formatted tier values of the input worker name
+   */
+  private static String getWorkerFormattedTierValues(Map<String, Map<String, String>> map,
+      String workerName) {
+    return map.values().stream().map((tierMap)
+        -> (String.format("%-14s", tierMap.getOrDefault(workerName, "-"))))
+        .collect(Collectors.joining(""));
+  }
+
+  /**
+   * Initializes member variables used to collect worker info.
+   */
+  private void initVariables() {
+    mSumCapacityBytes = 0;
+    mSumUsedBytes = 0;
+    mSumCapacityBytesOnTierMap = new TreeMap<>((a, b)
```

Comment:
Since this same  anonymous func is used multiple times, I would give it a name define a `Comparator` and use this Comparator instance to create `TreeMap`